From 937db8e0c74c7359d7cde1421e538227954cb2ea Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Fri, 25 Mar 2022 11:16:26 -0300
Subject: [PATCH 036/163] elf: Remove unused variables in tests

Checked on x86_64-linux-gnu and i686-linux-gnu.

(cherry picked from commit f60510ddafc62b81ed551b46db9761e408559322)
---
 elf/nodel2mod1.c  | 2 --
 elf/reldep8mod1.c | 2 --
 elf/reldep9mod1.c | 2 --
 3 files changed, 6 deletions(-)

diff --git a/elf/nodel2mod1.c b/elf/nodel2mod1.c
index acddc4cf8b..61d8c0edb0 100644
--- a/elf/nodel2mod1.c
+++ b/elf/nodel2mod1.c
@@ -9,9 +9,7 @@ void
 __attribute__((destructor))
 bar (void)
 {
-  static int i;
   foo ();
-  ++i;
 }
 void
 baz (void)
diff --git a/elf/reldep8mod1.c b/elf/reldep8mod1.c
index acddc4cf8b..61d8c0edb0 100644
--- a/elf/reldep8mod1.c
+++ b/elf/reldep8mod1.c
@@ -9,9 +9,7 @@ void
 __attribute__((destructor))
 bar (void)
 {
-  static int i;
   foo ();
-  ++i;
 }
 void
 baz (void)
diff --git a/elf/reldep9mod1.c b/elf/reldep9mod1.c
index 249a2bae1c..b75f9f1f1f 100644
--- a/elf/reldep9mod1.c
+++ b/elf/reldep9mod1.c
@@ -9,9 +9,7 @@ void
 __attribute__((destructor))
 bar (void)
 {
-  static int i;
   foo ();
-  ++i;
 }
 
 void
-- 
2.37.2

